extensions/scarb-doc/Cargo.toml

```diff
@@ -14,17 +14,21 @@ camino.workspace = true
 clap.workspace = true
 cairo-lang-compiler.workspace = true
 cairo-lang-defs.workspace = true
+cairo-lang-doc.workspace = true
+cairo-lang-filesystem.workspace = true
+cairo-lang-parser.workspace = true
 cairo-lang-semantic.workspace = true
+cairo-lang-starknet.workspace = true
 cairo-lang-syntax.workspace = true
-cairo-lang-filesystem.workspace = true
 cairo-lang-utils.workspace = true
+itertools.workspace = true
```

Comment:
Is this used here?
Same for `camino`.

Reply:
No idea tbh, I just changed the order, will check